k8s/charts/seaweedfs/Chart.yaml

```
@@ -2,4 +2,4 @@ apiVersion: v1
 description: SeaweedFS
 name: seaweedfs
 appVersion: "3.56"
-version: "3.56"
+version: "3.57"
```

```suggestion
version: "3.56"
```
Need to keep it in sync with the binary release version.